### DISCUSSION TITLE
In StaMPS, an error occurs when executing stamps(1,1), and ps_load_initial_isce generates an error message.

### DISCUSSION THREAD

**Discussion by imzhy666** [alexisInSAR/EZ-InSAR#92] [Help]
"How can I solve the problem where an error occurs in StaMPS when executing stamps(1,1), and ps_load_initial_isce generates an error message?"
![1](https://github.com/alexisInSAR/EZ-InSAR/assets/167092773/1f09f183-6a07-46ff-9b38-02ace5ba700d)
![2](https://github.com/alexisInSAR/EZ-InSAR/assets/167092773/cc8cb4ed-a778-493e-a10c-f0b6391fc132)
![3](https://github.com/alexisInSAR/EZ-InSAR/assets/167092773/749a976e-532c-4cc0-a66b-d85f9cb31c7f)

**Comment by alexisInSAR** [comment]
Hi, 

This issue can be linked to the compilation of StaMPS script. 

Kind regards, 

Alexis

**Comment by BitchSAR** [comment]
The pscands.1.ph file is not right. The value shouldn't have been zero.
